.github/actions/Tests/Unattended/pytest.ini

```diff
@@ -0,0 +1,3 @@
+[pytest]
+filterwarnings =
+    ignore:Unverified HTTPS request is being made.*
```

Comment:
Same

Reply:
This was a bad commit, I already deleted this folder.